lstchain/tools/lstchain_create_sampling_interval_coefficient_file.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from astropy.io import fits
+import h5py
+from tqdm import tqdm
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe_io_lst import LSTEventSource
+from ctapipe_io_lst.calibration import LSTR0Corrections
+from lstchain.calib.camera.sampling_interval_coefficient_calculate import SamplingIntervalCalculate
+from lstchain.paths import run_info_from_filename
+
+class SamplingIntervalCoefficientHDFWriter(Tool):
+    """
+    Tool that generates a sampling interval coefficient hdf file.
+    For getting help run:
+    lstchain_create_sampling_interval_coefficient_file --help
+    """
+
+    name = "SamplingIntervalCoefficientHDFWriter"
+    description = "Generate a sampling interval coefficient table"
+
+    input_fits = traits.Path(
+        help="Path to the generated peak count fits files",
+        directory_ok=True,
+    ).tag(config=True)
+
+    input_fits_hg = traits.Path(
+        help="Path to the generated fits sampling interval coefficient file for high gain",
+        directory_ok=False,
+    ).tag(config=True)
+
+    input_fits_lg = traits.Path(
+        help="Path to the generated fits sampling interval coefficient file for low gain",
+        directory_ok=False,
+    ).tag(config=True)
+
+    output = traits.Path(
+        help="Path to the output fits file",
+        directory_ok=False,
+    ).tag(config=True)
+
+    glob = traits.Unicode(
+        help="Filename pattern to glob files in the directory",
+        default_value="*.fits"
+    ).tag(config=True)
+
+    gain = traits.Int(
```

Comment:
Is there a reason to sparate HG and LG? 
It would be better to have all the data in a single file, so better to process them together if possible